lib/Net/ThreeScale/Client.pm

```diff
@@ -48,7 +48,7 @@ sub new {
 
 	my $self = {};
 	$self->{provider_key} = $params->{provider_key};
-	$self->{url}          = $params->{url} || 'http://su1.3scale.net';
+	$self->{url}          = $params->{url} || 'https://su1.3scale.net';
```

Comment:
Sounds good 👍 
We have seen major performance regressions with SSL without persistent connections. At least by order of magnitude and sometimes even more.